templates/components/carousel.html

```diff
@@ -10,7 +10,7 @@
         "accessibility": false
     }'>
     {{#each carousel.slides}}
-    <a href="{{url}}">
+    <a href="{{url}}" aria-label="{{#if this.alt_text.length '!=' 0}}{{this.alt_text}}{{else}}{{lang 'common.carousel_slide_number'}} {{ add @index 1 }}{{/if}}">
```

Comment:
Better to pass slide number into string template as param

Reply:
done